### PR TITLE
Add PROSP technical input tab

### PR DESCRIPTION
**Pull request by taustad**

### CODE REVIEW

File: backend/api/Controllers/PROSPController.cs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    public async Task<ProjectDto?> ImportFromSharepointAsync([FromQuery] Guid projectId,
+        [FromBody] SharePointImportDto[] dto)
+    {
+        try
+        {
+            var projectDto = await _prospSharepointImportService.ConvertSharepointFilesToProjectDto(projectId, dto);
+
+            return projectDto;
+        }
+        catch (Exception)
```
Comment (taustad) [inline backend/api/Controllers/PROSPController.cs:73]: Log the exception?

File: backend/api/Program.cs [hunk 2]
```diff
@@ -132,7 +137,10 @@
 
     options.ApplicationMode = true;
 });
-
+// builder.Services.AddMicrosoftIdentityWebApiAuthentication(config)
+//     .EnableTokenAcquisitionToCallDownstreamApi()
+//     .AddMicrosoftGraph()
+//     .AddInMemoryTokenCaches();
```
Comment (taustad) [inline backend/api/Program.cs:143]: Remove unused code?

File: backend/api/Controllers/PROSPController.cs [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        {
+            return null;
```
Comment (taustad) [inline backend/api/Controllers/PROSPController.cs:75]: Return project

File: backend/api/Services/ProspSharepointImportService.cs [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+    public Dictionary<string, bool> MapAssets(bool surf, bool substructure, bool topside, bool transport)
+    {
+        var assets = new Dictionary<string, bool>()
+                {
+                    {nameof(Surf), false},
+                    {nameof(Topside), false},
+                    {nameof(Substructure), false},
+                    {nameof(Transport), false},
+                };
```
Comment (taustad) [inline backend/api/Services/ProspSharepointImportService.cs:65]: `                    {nameof(Surf), surf},
                    {nameof(Topside), topside},
                    {nameof(Substructure), substructure},
                    {nameof(Transport), transport}`

File: frontend/src/Components/EditTechnicalInput/PROSPCaseList.tsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+            })
+            setProspCases(prosp)
+        })()
+    }, [project])
+
+    const GenerateWellTableRows = () => {
+        const tableRows: JSX.Element[] = []
+        project.cases.forEach((caseItem) => {
+            tableRows.push((
+                <PROSPTableRow key={caseItem.id!} project={project} prospCases={prospCases} setProspCases={setProspCases} caseId={caseItem.id!} driveItems={driveItems} />
```
Comment (taustad) [inline frontend/src/Components/EditTechnicalInput/PROSPCaseList.tsx:41]: <PROSPTableRow
                    key={caseItem.id!}
                    project={project}
                    prospCases={prospCases}
                    setProspCases={setProspCases}
                    caseId={caseItem.id!}
                    driveItems={driveItems}
                />

**Review by ahmedoabdi — APPROVED** [review]
LGTM!

**Review by ahmedoabdi — APPROVED** [review]

**Pull request merged**